src/NuGet.Core/NuGet.Build.Tasks/NuGet.Build.Tasks.csproj

```diff
@@ -56,6 +66,24 @@
     </EmbeddedResource>
   </ItemGroup>
 
-  <Import Project="$(BuildCommonDirectory)common.targets"/>
+  <ItemGroup Condition="$(DefineConstants.Contains(SIGNED_BUILD))">
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+      <_Parameter1>NuGet.Build.Tasks.Console, PublicKey=0024000004800000940000000602000000240000525341310004000001000100b5fc90e7027f67871e773a8fde8938c81dd402ba65b9201d60593e96c492651e889cc13f1415ebb53fac1131ae0bd333c5ee6021672d9718ea31a8aebd0da0072f25d87dba6fc90ffd598ed4da35e44c398c454307e8e33b8426143daec9f596836f97c8f74750e5975c64e2189f45def46b2a2b1247adc3652bf5c308055da9</_Parameter1>
```

Comment:
We don't use InternalsVisibleTo between [shipping assemblies](https://github.com/NuGet/NuGet.Client/pull/3180/files#diff-85d4cc615fcaaf78685c6435fc0f6de2R158)

Reply:
Okay I'll make the classes public